src/components/static/ConnectLine.js

```diff
@@ -0,0 +1,40 @@
+import { default as EventEmitter } from "../../event/eventEmitter";
+import { motion as m } from "framer-motion";
+import { useMemo, useState } from "react";
+
+import "./../../styles/ConnectLine.css";
+
+export default function ConnectLine() {
+  const [isSocketConnect, setIsSocketConnect] = useState(false);
+  EventEmitter.subscribe("setConnectStatus", (v) => setIsSocketConnect(v));
```

Comment:
This needs to be mooved to some other place. Otherwise, it will subscribe every time a ConnectLine is rendered

Reply:
Normally, a ConnectLine should receive a connection state via props.

Reply:
e.g. in src/index.js